addons/binding/org.openhab.binding.fritzaha/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
when refactoring the namespace, you missed to change the folder name itself - this is still fritzaha.